src/user_account/models.py

```diff
@@ -12,4 +13,7 @@ class User(AbstractUser):
     birth_date = models.DateField(null=True, blank=True, default=datetime.date.today)
 
     def update_score(self):
-        self.avr_score = self.test_results.values('avr_score').last().get('avr_score')
+        qs = self.test_results.values('avr_score').annotate(
+            points=Sum('avr_score')
+        )
+        self.avr_score = sum(int(entry['avr_score']) for entry in qs)
```

Comment:
Looks like what you doing here is summing up all scores:
```
self.avr_score = self.test_results.aggregate(points=Sum('avr_score')).get('points', 0.0)
```

Reply:
Yes, it was intended.
Suppose first I calculated the average score for passing the test, then we summarize the average points for each test passed. But now I realized that this approach is wrong, I still need to divide by the number of passed tests, so it will be more interesting.

Reply:
Fixed.